packages/admin/admin/src/stack/StackLink.tsx

```diff
@@ -3,12 +3,12 @@ import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-d
 
 import { IStackSwitchApi, useStackSwitchApi } from "./Switch";
 
-interface StackLinkProps extends Omit<RouterLinkProps, "to"> {
+type StackLinkProps = {
```

Comment:
Otherwise, I get following error for some reason:

<img width="1493" alt="Bildschirmfoto 2024-01-12 um 14 59 50" src="https://github.com/vivid-planet/comet/assets/13380047/b07a1ab6-e845-4196-9d40-73d2570d651e">

Reply:
Weird, but nice fix 😁